draft-ietf-taps-interface.md

```diff
@@ -1306,6 +1312,126 @@ Properties will include different information:
   properties can be derived from the local provisioning domain {{RFC7556}},
   measurements by the Protocol Stack, or other sources.
 
+
+### Notification of excessive retransmissions {#conn-retrans-notify}
+
+Type:
+: Boolean
+
+This property specifies whether an application considers it useful to be informed in case sent data was retransmitted more often than a certain threshold.
+When set to true, the effect is twofold:
+The application may receive events in case excessive retransmissions.
+In addition, the transport system considers this as a preference to use transports stacks that can provide this notification. This is not a strict requirement.
```

Comment:
This basically says that this property is two things at the same time:
1. A Connection Property saying that the application wants these notifications in case the transport stack can provide them,
2. A Selection Property to "Prefer" transport stacks that can provide these notifications.

I'm wondering if this property really should be both in one.
I think this text got written before we had an actual concept of Preferences in Selection Properties. Maybe we want to make this an actual Selection Property instead, with data type "Preference"? And then once the Connection is instantiated, it becomes a Connection property like all other Selection Properties, providing a Boolean of whether the feature is there or not?
I'd also be okay with it staying like this for this PR though.